lib/fitfree.gi

```diff
@@ -1398,6 +1398,31 @@ local l;
   fi;
 end);
 
+
+#############################################################################
+##
+#M  SylowSubgroupOp( <G>, <pi> )
+##
+## Fitting free approach
+##
+InstallMethod( SylowSubgroupOp, "fitting free",true,
+    [ IsGroup and CanComputeFittingFree,IsPosInt ],
+    -1, # rank lower than anything that would imply CanComputeFittingFree
+function(G,pi)
+local l;
+  if Set(Factors(Size(G)))=[pi] then 
```

Comment:
Optional suggestion: could also do this:
```suggestion
  if PrimeDivisors(Size(G))=[pi] then 
```

Reply:
Yes, one could. But neither runtime cost, nor size of code makes a big difference.